php/8.2.dockerfile

```diff
@@ -13,6 +13,12 @@ ENV PHP_VERSION=${PHP_VERSION} \
     PHP_MAX_EXECUTION_TIME=30 \
     PHP_MAX_INPUT_VARS=1500 \
     PHP_ASSERT=-1 \
+    PHP_POST_MAX_SIZE=100M \
+    PHP_UPLOAD_MAX_FILESIZE=100M \
+    PHP_OPCACHE_ENABLE_FILE_OVERRIDE=0 \
+    PHP_OPCACHE_INTERNED_STRINGS_BUFFER=8 \
+    PHP_DETECT_UNICODE=1 \
+    PHP_REALPATH_CACHE_TTL=120 \
```

Comment:
Wouldn't it be better to have a higher default value for production container?
We are using immutable container so realpath did not change during runtime.